lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -190,6 +185,7 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
   }
 
   Widget handleUploadWidget(BuildContext context, [Success success]) {
+    print(success);
```

Comment:
please remove this

Reply:
It's not working right now, need this to test :(